getting-started/002-search.md

```diff
@@ -73,7 +74,7 @@ Note that all the results reside within Great Britain:
 * YMCA, Lenton Abbey, Nottinghamshire
 * YMCA, Old Clee, Lincolnshire
 
-Now you can try the same search request with different country codes and see the results change.
+If you attempt the same search request with different country codes, the results change to reflect YMCA locations within the country.
```

Comment:
`the country` -> `that region`

sorry, sounds super pedantic, but following on from using `GBR` as our example it's not really correct to call them 'country' codes as that one technically refers to the island which contains (most of) the lands of England, Northern Ireland, Scotland and Wales, I'll shut up now.